draft-irtf-cfrg-hybrid-kems.md

```diff
@@ -128,24 +139,50 @@ informative:
-security properties.
+"Post-quantum" (PQ) algorithms are designed to resist attack by a quantum computer,
+in contrast to "traditional" algorithms.  However, given the novelty of PQ
+algorithms, there is some concern that PQ algorithms currently believed to be
+secure will be broken.  Hybrid constructions that combine both PQ and
+traditional algorithms can help moderate this risk while still providing
+security against quantum attack. In this document, we define constructions for
+hybrid Key Encapsulation Mechanisms (KEMs) based on combining a traditional KEM
+and a PQ KEM. Hybrid KEMs using these constructions provide strong security
+properties as long as the undelying algorithms are secure.
 
 --- middle
 
 # Introduction {#intro}
 
-There are many choices that can be made when specifying a hybrid KEM: the
-constituent KEMs; their security levels; the combiner; and the hash within,
-to name but a few. Having too many similar options are a burden to the
-ecosystem.
-
-The aim of this document is provide a small set of techniques for
-constructing hybrid KEMs designed to achieve specific security properties
-given conforming component algorithms, that should be suitable for the vast
-majority of use cases.
+Post-quantum (PQ) algorithms offer a redesign of traditional algorithms tailored
+towards resisting attack from a quantum computer. Key Encapsulation Mechanisms
+(KEMs), are a standardized algorithm type that can be used to build protocols in
+lieu of traditional, quantum-vulnerable variants such as Diffie-Hellman (DH)
+based protocols.  Upgrading protocols to use PQ KEMs is a priority for the
+protocol design community, due to the possibility of "harvest now, decrypt
+later" attacks.
+
+Given the novelty of PQ algorithms, however, there is some concern that PQ
+algorithms currently believed to be secure will be broken.  Hybrid
+constructions that combine both PQ and traditional algorithms can help moderate
+this risk while still providing security against quantum attack.  If construted
+properly, a hybrid KEM will retain the properties of either constituent KEM
+even if the other KEM is compromised.  If the PQ KEM is broken by new
+cryptanalysis, then the hybrid KEM should continue to provide security against
+non-quantum attackers by virtue of its traditional KEM component.  If the
+traditional KEM is broken by a quantum computer, then the hybrid KEM should
+continue to resist quantum attack by virtue of its PQ KEM component.
+
+In addition to guarding against algorithm flaws, this property also guards
+against flaws in implementations, such as timing attacks.  Hybrid KEMs can also
+facilitate faster deployment of PQ security by allowing applications to
+incorporate PQ algorithms while still meeting compliance requirements based on
+traditional algorithms.
+
+In this document, we define constructions for hybrid KEMs based on combining a
+traditional KEM and a PQ KEM.  The aim of this document is provide a small set
```

Comment:
But not only KEMs as components, as in QSF - we combine traditional components and PQ KEMs